recipes/fipy/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "FiPy" %}
+{% set version = "3.1.3" %}
+{% set sha256 = "88d72fea482492634696e7f8705c604c138264b64765ca4dd62245360f704f4b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - 2to3 --write .  # [py3k]
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - gmsh <4.0   # [not (win and (py2k or x86))]
+    - pysparse    # [not py3k]
+    - pytrilinos  # [not win and not py3k]
+    - mpi4py      # [not win and not py3k]
+    - matplotlib
+    - mayavi
+    - weave       # [not py3k]
+
+test:
+  imports:
+    - fipy
+
+about:
+  home: http://www.ctcms.nist.gov/fipy
+  license: NIST public domain
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.rst'
```

The `RECIPE_DIR` part isn't needed
```suggestion
  license_file: LICENSE.rst
```